NOTES-WINDOWS.md

```diff
@@ -148,6 +148,8 @@ Clang-based compiler (`bcc32c.exe`) in v10.3.3 Community Edition.
 
  3. Go to the root of the OpenSSL source directory and run:
     `perl Configure BC-32 --prefix=%CD%`
+    for Win64 build use:
```

Comment:
I think it's more natural to say
```suggestion
    for Win64 builds use:
```

Reply:
fix it here https://github.com/openssl/openssl/pull/23377/commits/ed9e258e2e10dad91e43bbc342b73d35d66b5591